### PR TITLE
GitHub issue template

### PR DESCRIPTION
**Pull request by Fecony**
## Description
When I opened issue for git initialization with CLI `--no-git` flag I saw blank issue and didn't know how to write what I wanted to add. This PR will allow users to add new issue for `bug report` / `documentation bug(improvement)` / `feature request` templates. They can be improved to collect more information from user, or separated for each package `cli feature` or `cli bug` for example.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## Expected behavior
+A clear and concise description of what you expected to happen (or code).
+
+## Possible Solution
+<!--- Only if you have suggestions on a fix for the bug -->
+
+## Additional information
+
+#### Environment
+- Node/npm version: [e.g. Node 8/npm 5]
```
Comment (NickSeagull) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: I'd add also the Booster version:
```suggestion
- Booster version: [e.g. 0.8.2] (You can check this with `boost version`
- Node/npm version: [e.g. Node 8/npm 5]
```

**Review by NickSeagull — COMMENTED** [review]
Looks awesome! Just one small comment

**Comment by Fecony** [comment]
@NickSeagull Added booster version to issue template

**Comment by NickSeagull** [comment]
@boostercloud/booster-core I'm gonna merge this as they are only docs and it is super useful for possible future contributions

**Pull request merged**